.github/workflows/get-streamers.yml

```diff
@@ -1,7 +1,6 @@
 name: "Get Streamers"
 on:
-  schedule:
-#     - cron: "*/10 * * * *"
+  workflow_dispatch
```

Comment:
I think workflow_dispatch can even stay there once it all works - to trigger it if needed